src/journal/journald-server.c

```diff
@@ -818,6 +818,78 @@ static bool shall_try_append_again(JournalFile *f, int r) {
         }
 }
 
+typedef struct FailedEntry {
+        int r;
```

Comment:
We use `r` as the local variable name for errors, but in structs, we should use something more descriptive, so can we call this `error`?